lib/NewDataList/NewDataList.js

```diff
@@ -82,7 +82,7 @@ module.exports = kind({
 		// potentially not in view, and focusing would cause it to
 		// scroll it into view for no reason apparent to the user.
 		// Instead, we just focus the first visible list element.
-		if (event.focusType === 'default' && event.index) {
+		if (event.focusType === 'default' && typeof event.index === 'number') {
```

Comment:
Doing some really simple JSPerf-esque experiments in the console (Mac Chrome and TV) seems to indicate that the performance of these checks is essentially the same.